source/hardware/controllers/pioneer_ddj_400.rst

```diff
@@ -0,0 +1,185 @@
+6     :hwlabel:`shift` + :hwlabel:`IN/-4BEAT`                  Toggle :hwlabel:`IN ADJUST` mode on active loop. Adjust the loop's start position using the jog wheel.
+7     Jog Wheel (top)                                          Scratch (move play position)
+7     Jog Wheel (outer)                                        Nudge tempo up or down temporarily
+8     :hwlabel:`HOT CUE` mode                                  Sets pads to hot cue mode. *TODO: See using hotcues*
+9     :hwlabel:`BEAT LOOP` mode                                Sets pads to beat loop mode. *TODO: See using beat loop*
+10    :hwlabel:`BEAT JUMP` mode                                Sets pads to beat jump mode. *TODO: See using beat jump*
+11    :hwlabel:`SAMPLER` mode                                  Sets pads to sampler mode. *TODO: See using sampler*
+12    :hwlabel:`TEMPO` slider                                  Adjust playback speed
+13    Performance pads                                         Functionality differs depending on the pad mode selected by 8-11 *See pad mode descriptions*
+14    :hwlabel:`PLAY/PAUSE`                                    Play/pauses a track
+15    :hwlabel:`CUE`                                           Sets cue point if paused. Return to cue point and stop if playing. Play while held on the set cue point.
+====  =======================================================  ======================================================================
+
+Mixer section (p. 8)
+~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_mixer.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (mixer section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (mixer section)
+
+====  =======================================================  ======================================================================
+No.   Control                                                  Function
+====  =======================================================  ======================================================================
+1     :hwlabel:`TRIM`                                          Adjusts channel output gain
+2     :hwlabel:`MASTER`                                        Adjusts master output level
+3     Headphone :hwlabel:`CUE` (Master)                        Sends master output to headphones (adjust with headphone mixing knob)
+4     EQs (:hwlabel:`LOW` :hwlabel:`MID` :hwlabel:`HIGH`)      Boosts or cuts channel frequencies
+5     Channel level indicators                                 Displays channel levels (before being sent through channel faders)
+6     :hwlabel:`HEADPHONES` :hwlabel:`MIXING`                  Adjusts between headphone :hwlabel:`CUE` and :hwlabel:`MASTER` levels sent to headphones
+7     :hwlabel:`FILTER`                                        Turn left to apply low pass filter to channel, right to apply high pass filter
```

Comment:
```suggestion
7     :hwlabel:`FILTER`                                        QuickEffect superknob (by default controls a Filter, but this can be changed in the :ref:`preferences <preferences-equalizers>`)
```